src/mapGenerator/mapgenManager.py

```diff
@@ -0,0 +1,156 @@
+# system imports
+import logging
+import sys
+from PyQt5 import QtCore, QtGui, QtWidgets, QtNetwork
+import os
+import re
+import shutil
+import random
+
+# local imports
+from config import Settings
+import fafpath
+from vault.dialogs import downloadFile
+from mapGenerator.mapgenProcess import MapGeneratorProcess
+from fa.maps import getUserMapsFolder
+
+logger = logging.getLogger(__name__)
+
+class MapGeneratorManager(object):
+    def __init__(self):
+        self.releaseUrl = "https://github.com/FAForever/Neroxis-Map-Generator/releases/download/{}/NeroxisGen_{}.jar"
+        self.latestRelease = "https://github.com/FAForever/Neroxis-Map-Generator/releases/latest"
```

Comment:
is there no way to define those as constant, non object/instance, variables?

Reply:
also, don't duplicate the address, derive both of them by combining them with a third base address.